python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -1576,6 +1606,13 @@ def is_multi_gpu(self):
         """
         return True
 
+    def is_remote(self):
```

Comment:
This seems like a requirement from higher-level abstraction that doesn't apply generally to PG.  Is there a way we can avoid having this method?

Reply:
should be fixed now